.gitignore

```diff
@@ -5,4 +5,5 @@ config.json
 !.yarn/releases
 !.yarn/plugins
 !.yarn/sdks
-!.yarn/versions
\ No newline at end of file
+!.yarn/versions
+node_modules
```

Comment:
why is node_modules being added? this project should never have a node_modules

Reply:
wait

Reply:
sorry it was because i was installing the dependencies locally, will fix

Reply:
All good